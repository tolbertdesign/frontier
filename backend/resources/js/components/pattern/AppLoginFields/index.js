import AppLoginFields from './AppLoginFields.vue'

export default AppLoginFields
