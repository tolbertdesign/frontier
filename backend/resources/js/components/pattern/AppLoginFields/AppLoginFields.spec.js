import { shallowMount, createLocalVue } from '@vue/test-utils'
import AppLoginFields from './AppLoginFields.vue'

const localVue = createLocalVue()

const emailAddressLabel = 'email address label'
const passwordLabel = 'password label'
const rememberMeLabel = 'remember me label'
const showPasswordLabel = 'show password label'
const error = 'error'

describe('AppLoginFields', () => {
  it('Snapshop test for AppLoginFields', () => {
    const wrapper = shallowMount(AppLoginFields, {
      localVue,
      propsData: {
        emailAddressLabel: emailAddressLabel,
        passwordLabel: passwordLabel,
        rememberMeLabel: rememberMeLabel,
        showPasswordLabel: showPasswordLabel,
        error: error,
      },
    })
    expect(wrapper).toMatchSnapshot()
  })
})
