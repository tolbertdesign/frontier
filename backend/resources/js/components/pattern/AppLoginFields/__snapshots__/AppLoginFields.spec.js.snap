// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLoginFields Snapshop test for AppLoginFields 1`] = `
<div>
  <div class="form-group"><label for="email" class="sr-only">

    </label> <input type="email" name="email" required="required" class="is-invalid form-control mw-250px mx-auto"></div>
  <div class="form-group"><label for="email" class="sr-only">

    </label> <input type="password" name="password" required="required" class="is-invalid form-control mw-250px mx-auto"></div>
  <div class="d-flex mw-250px mx-auto mb-4">
    <div class="checkbox text-left mx-auto"><label><input type="checkbox" name="remember">
      </label></div>
    <div class="show-password text-right mx-auto"><a href="javascript:void(0);">

      </a></div>
  </div>
</div>
`;
